app/data/news.js

```diff
@@ -9,6 +9,12 @@ export default [
     content:
       'We received <strong>over 300 proposals</strong> for JSHeroes 2019! We will <strong>announce</strong> all the speakers by the 1st of February!',
   },
+  {
+    title: 'Sticker Contest',
+    content: `<div>Announcing: JSHeroes Sticker Contest</div>
+              <div>Enter the sticker design competition to win cool stuff or just to have fun!</div>
+              <div>Find all the <a href="/sticker-contest">details here.</Link></div>`,
```

Comment:
Noticed: `<a href="/sticker-contest">details here.</Link>` should be either Link or anchor.

Reply:
fixed